src/contrib/cluster/Akka.Cluster.Sharding/Delivery/Internal/ShardingConsumerControllerImpl.cs

```diff
@@ -128,6 +141,19 @@ private void Active()
                 }
             }
         });
+        
+        ReceiveAny(msg =>
+        {
+            if (Settings.AllowBypass)
+            {
+                _consumer.Tell(msg);
+            }
+            else
+            {
+                _log.Warning($"Message unhandled [{msg}]. If you need to pass this message to the consumer sharding entity actor, set \"akka.reliable-delivery.sharding.consumer-controller.allow-bypass\" to true");
```

Comment:
Log a warning when a message was unhandled, informing the user that they can allow this message to be forwarded if they set `allow-bypass` to true